src/types/categories/interfaces/categories.interfaces.ts

```diff
@@ -1,5 +1,15 @@
+import { CommonEntityFields } from '~/types/common/common.index'
 import { RequestParams } from '~/types/services/types/services.types'
 
 export interface CategoriesParams extends RequestParams {
   name: string
 }
+
+export interface CreateCategoriesParams {
+  name: string
```

Comment:
maybe better to pick it from Categories or you can do like this: 
name: Categories['name']

Reply:
> maybe better to pick it from Categories or you can do like this: name: Categories['name']

Yup, that's better. Fixed